chromium_src/third_party/blink/renderer/bindings/modules/v8/v8_navigator.cc

```diff
@@ -1,10 +1,10 @@
 /* Copyright (c) 2021 The Brave Authors. All rights reserved.
  * This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
- * You can obtain one at http://mozilla.org/MPL/2.0/. */
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
 
 #include "third_party/blink/renderer/platform/bindings/idl_member_installer.h"
 
-#define InstallAttributes BraveInstallAttributes<V8Navigator>
+#define InstallAttributes BraveInstallAttributes<bool>
```

Comment:
maybe create a specific type to make it clearer? `bool` doesn't make sense here.

for example: https://gist.github.com/goodov/9ed098cd726a573e90a9987e1a6b872a

Reply:
Thanks @goodov, I thought it would be a bit of an overkill to declare a type just for this, but might as well.

Updated via [5f9c3f5](https://github.com/brave/brave-core/commit/5f9c3f5ef1f39a802a705c632b132dea4858678a)